Shadan/style.css

```diff
@@ -0,0 +1,159 @@
+*
+{
+    margin:0;
+    padding:0;
+}
+html{
+    background-color: #f7f7f7;
+}
+header{
+    background-color: #e6e6e6;
+
+}
+
+header .topcenter{
+    position:relative;
+}
+
+nav .logo{
+
+    width:50px;
+    height:50px;
+   position: absolute;
+   left:25%;
+}
+header nav .navbar{
+    margin-top:20px;
+    text-align: center;
+     position: absolute;
+     right:25%;
+     
+}
```

Comment:
I don't think that this is the best practice for the navbar, you can have a container div as parent and give it `width:60%` maybe and `margin: auto`and here you can use `float: left` for the logo and `float:right` for the buttons